src/styles/blocks/body.scss

```
@@ -0,0 +1,117 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
```

```suggestion
* {
  box-sizing: border-box;
```
Don't reset margins using * selector. Specify the exact tags you want to reset